e2e_tests/tests/model_hub/test_transformers.py

```diff
@@ -39,3 +39,233 @@ def test_token_classification_ner_amp() -> None:
     config = set_docker_image(config)
 
     exp.run_basic_test_with_temp_config(config, example_path, 1)
+
+
+@pytest.mark.model_hub  # type: ignore
+@pytest.mark.distributed  # type: ignore
+def test_language_modeling_clm() -> None:
+    example_path = conf.model_hub_examples_path("huggingface/language-modeling")
+    config = conf.load_config(os.path.join(example_path, "clm_config.yaml"))
+    config = conf.set_max_length(config, {"batches": 200})
```

Comment:
Can you confirm that for each of these examples 200 batches runs at least one train step and one validation step?

Reply:
yes, we always perform a validation step once `max_length` is reached.